<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorizeSurfaceOptions</class>
 <widget class="QWidget" name="ColorizeSurfaceOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioButtonCurrentFrame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>C&amp;urrent frame</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButtonAllFrames">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>A&amp;ll frames</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame sampling</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBoxFrameSampling">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="buttonCompute">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Color display</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboBoxColorDisplay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>mean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>count</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxRemoveOccluded">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remove occluded surface color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Occlusion threshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxOcclusionThreshold">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Points at higher depth difference are considered occluded</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxRemoveMasked">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remove masked surface color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
